tensorflow/python/tools/import_pb_to_tensorboard.py

```diff
@@ -41,19 +41,19 @@
 
 
 def import_to_tensorboard(model_dir, log_dir, tag_set):
-  """View an imported protobuf model (`.pb` file) as a graph in Tensorboard.
+  """View an SavedModel as a graph in Tensorboard.
```

Comment:
imported/saved protobuf model (`.pb` file)

Reply:
@ashutosh1919 Thank you for your suggestion, however, this function will try to load model_dir as SavedModel (expected `saved_model.pb` or `saved_mode.pbtxt` inside it), not a protobuf model (.pb file).